theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-contribution.ts

```diff
@@ -257,7 +257,7 @@ export class TraceViewerContribution
                 try {
                     await this.traceServerConfigService.stopTraceServer();
                     this.messageService.info('Trace server terminated successfully.');
-                    TraceServerConnectionStatusService.renderStatus(false);
+                    TraceServerConnectionStatusClientImpl.renderStatus(false);
                 } catch (err) {
                     this.messageService.error('Failed to stop the trace server.');
                 }
```

Comment:
I get two compilation errors locally, in this file, if I rebase this PR on current latest master tip. This is the error I get at both lines 97 and 166 (current master version):

```
Cannot find name 'TraceServerConnectionStatusService'.
TraceServerConnectionStatusService.renderStatus(true);
```

Reply:
Fixed it with the latest changes.